content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -273,6 +273,33 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
 {{% /tab %}}
 {{< /tabs >}}
 
+### Verifying setup
+
+To verify you've configured the APM integration with RUM, follow the steps below based on the SDK you installed RUM with.
+
+
+{{< tabs >}}
+{{% tab "Browser" %}}
+
+1. Visit a page in your application.
+2. In your browser's developer tools, go to the **Network** tab.
+3. Check the request headers for a resource request that you expect to be correlated contains the [correlation headers from Datadog][1].
+
+[1]: /real_user_monitoring/connect_rum_and_traces?tab=browserrum#how-are-rum-resources-linked-to-traces
+
+{{% /tab %}}
+
+{{% tab "Mobile" %}}
+
+1. Visit a page in your application.
+2. In your browser's developer tools, go to the **Network** tab.
+3. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][1].
+
+[1]: /real_user_monitoring/connect_rum_and_traces?tab=browserrum#how-are-rum-resources-linked-to-traces
```

Comment:
This whole section doesn't make sense for Mobile Apps. I'm guessing the Verifying setup instructions are going to be different for each platform. E.g. for Android : 

---

1. Run your application form Android Studio
2. Visit a screen in your application.
3. Open Android Studio's [**Network Inspector**][1].
4. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][2].

[1]: https://developer.android.com/studio/debug/network-profiler#network-inspector-overview
[2]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=androidrum#how-are-rum-resources-linked-to-traces

---

Reply:
Hi @xgouchet, I had a feeling that would be case, so I'm glad I asked! Would you / @DataDog/rum-mobile be able to help fill in the blanks for the remaining SDKs, as I don't have much insight into those? I did my best guess for iOS.

Reply:
I'll be PTO from tomorrow until August 7th, but I'll pass the word around ;)

Reply:
@xgouchet That works - thanks very much! :D